app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -92,6 +92,7 @@ open class TelemetryIntegration protected constructor(
         val TILE_CUSTOM = "custom"
         val POCKET_VIDEO_MEGATILE = "pocket_video_tile"
         val YOUTUBE_TILE = "youtube_tile"
+        val SESSION_WITHOUT_YOUTUBE = "session_without_youtube"
```

Comment:
I talked to Su (who is doing metrics for this) and I think that this doesn't need to be collected - he said it's easy enough to group by sessions that don't have a youtube_tile event, so can we remove this commit? (and the telemetry docs for it)